### PR TITLE
Reference individual contracts packages

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 4.3
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Follows #31567

### CODE REVIEW

File: src/Symfony/Contracts/phpunit.xml.dist [hunk 1]
```diff
@@ -15,6 +15,7 @@
     <testsuites>
         <testsuite name="Symfony Contracts Test Suite">
             <directory>./Tests/</directory>
+            <directory>./Translation/Test/</directory>
```
Comment (ro0NL) [inline src/Symfony/Contracts/phpunit.xml.dist]: shouldnt we move all tests to `src/.../Tests/` (note plural).. what happened? :sweat_smile: that's what all components do.

Comment (nicolas-grekas) [reply src/Symfony/Contracts/phpunit.xml.dist]: reusable tests are in `Test`, tests themselves in `Tests`
TranslatorTest is used in the Translation component.

**Comment by nicolas-grekas** [comment]
Good to go, remaining failure will be fixed once this is merged and splitted.

**Review by derrabus — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**